client/src/sagas/types.js

```diff
@@ -1,4 +1,12 @@
+// TODO: probably break this up into more specialized files
```

Comment:
Yeah, I need a better system for this. 💡 🔮 maybe I should export functions which return action objects from this file and sort them by type (auth, alert, etc.)